demo-shell/src/app/components/file-view/file-view.component.html

```diff
@@ -4,7 +4,11 @@
         <adf-info-drawer [title]="'APP.INFO_DRAWER.TITLE' | translate">
 
             <adf-info-drawer-tab label="APP.INFO_DRAWER.COMMENTS">
-                <adf-comments [nodeId]="nodeId" [readOnly]="!isCommentEnabled"></adf-comments>
+                <adf-node-comments 
```

Comment:
you might also need upgrading this in ACA? https://github.com/Alfresco/alfresco-content-app/blob/develop/app/src/app/components/info-drawer/comments-tab/comments-tab.component.ts#L32